Fixed issue #59.  Changed the create team page into a dialog window.

cdterry87 opened the pull request:
Also removed the old way of creating teams.

chamerling commented:
@cdterry87 dialog remains displayed after creation on all pages. Please clean the state

![Capture d’écran 2019-10-06 à 11 42 47](https://user-images.githubusercontent.com/264403/66267195-87714d80-e82e-11e9-9469-dbf16869424e.png)

cdterry87 commented:
@chamerling I am now handling the state of the Create Team dialog in state and clearing it out after a team is created.  Check it out now and let me know if there are any issues.

chamerling commented on src/App.vue:
```diff
@@ -148,6 +148,19 @@
+                <v-toolbar-title class="white--text">Create a new team</v-toolbar-title>
+            </v-toolbar>
+            <v-card-text>
+              <CreateTeam />
```
All the things with vuex state to manage `createTeamDialog` is too complex. You should just be able to manage local state in this App component. Add an event listener to `CreateTeam` which emits event when form is submitted and then hide the dialog from current component

```html
<CreateTeam @close="dialog = false"/>
```

chamerling commented on src/components/CreateTeam.vue:
```diff
@@ -0,0 +1,123 @@
+        this.$store.dispatch("setCreateTeamDialog", false);
+        this.$router.push({ name: "team", params: { name: team.name } });
+
+        this.clear();
```
Then here 

```js
this.$emit("close")
```

cdterry87 commented:
@chamerling I was definitely overthinking this, lol.  I implemented your solution, and it is working great now.  I removed everything related to the dialog from the vuex store.  Thank you for the feedback and the help getting this resolved!  Let me know if there is anything else I can do to help.

chamerling commented:
@cdterry87 great!
